diff -urN dconf-0.34.0/common/meson.build dconf-0.34.0.new/common/meson.build
--- dconf-0.34.0/common/meson.build	2019-09-09 18:56:47.000000000 -0400
+++ dconf-0.34.0.new/common/meson.build	2019-11-15 16:08:07.908760276 -0500
@@ -28,7 +28,7 @@
 
 libdconf_common_dep = declare_dependency(
   dependencies: glib_dep,
-  link_whole: libdconf_common,
+  link_with: libdconf_common,
 )
 
 libdconf_common_hidden = static_library(
