diff -urN gdm-3.38.1/data/pam-exherbo/gdm-launch-environment.pam gdm-3.38.1-fixed/data/pam-exherbo/gdm-launch-environment.pam
--- gdm-3.38.1/data/pam-exherbo/gdm-launch-environment.pam	2020-10-13 13:15:20.078997600 -0400
+++ gdm-3.38.1-fixed/data/pam-exherbo/gdm-launch-environment.pam	2020-11-03 04:32:09.869527580 -0500
@@ -8,7 +8,7 @@
 
 password    required    pam_deny.so
 
--session    optional    pam_systemd.so
+-session    optional    pam_elogind.so
 session     optional    pam_keyinit.so force revoke
 session     required    pam_succeed_if.so audit quiet_success user = gdm
 session     required    pam_permit.so
